simpleclient_pushgateway/src/main/java/io/prometheus/client/exporter/PushGateway.java

```diff
@@ -54,15 +56,42 @@
  */
 public class PushGateway {
 
-  private final String address;
+  // Visible for testing
```

Comment:
There's no way to explicitly say package, so protected is the next best thing.

This avoids misconceptions that you forgot to put in an access level.